src/core/src/op/random_uniform.cpp

```diff
@@ -92,6 +92,10 @@ bool RandomUniform::constant_fold(OutputVector& output_values, const OutputVecto
     return false;
```

Comment:
Can be removed as Base implementation will return false.

Reply:
done